game/graphics/mdlvisual.cpp

```diff
@@ -591,9 +591,9 @@ const Animation::Sequence* MdlVisual::startAnimAndGet(Npc& npc, AnimationSolver:
     auto inter = npc.interactive();
     const Animation::Sequence *sq = solver.solveAnim(inter,a,*skInst);
     if(sq!=nullptr){
-      if(skInst->startAnim(solver,sq,comb,BS_MOBINTERACT,Pose::NoHint,npc.world().tickCount())) {
+      auto bs=sq->name.find("T_LADDER")==0 ? BS_CLIMB : BS_MOBINTERACT;
```

Comment:
There is no need to inspect animation names. Not to mention they are unreliable. 
`inter->isLadder()`

Reply:
`isLadder()` gave an incomplete class error and i didn't know how to deal with it. Should have stated that in the first place. Including the header fixed it.

Reply:
> isLadder() gave an incomplete class error and i didn't know how to deal with it.

Can you show error itself?

Reply:
```
/home/tt/git/OpenGothic/game/graphics/mdlvisual.cpp:594: error: invalid use of incomplete type ‘class Interactive’
/home/tt/git/OpenGothic/game/graphics/mdlvisual.cpp: In member function ‘const Animation::Sequence* MdlVisual::startAnimAndGet(Npc&, AnimationSolver::Anim, uint8_t, WeaponState, WalkBit)’:
/home/tt/git/OpenGothic/game/graphics/mdlvisual.cpp:594:20: error: invalid use of incomplete type ‘class Interactive’
  594 |       auto bs=inter->isLadder() ? BS_CLIMB : BS_MOBINTERACT;
      |                    ^~
```

Reply:
`#include "interactive.h"`